src/index.js

```diff
@@ -6,6 +6,9 @@ let joinButtonAction = onJoin;
 getCurrentUser()
 getQueryParams()
 
+document.body.style.height = window.innerHeight + 'px';
+document.body.style.width = window.innerWidth + 'px';
+
```

Comment:
Can we set these styles via css, so that the site also looks good when Javascript is disabled?